doc/developers/contributing.rst

```diff
@@ -1325,6 +1325,27 @@ interface might be that you want to use it together with model evaluation and
 selection tools such as :class:`model_selection.GridSearchCV` and
 :class:`pipeline.Pipeline`.
 
+Setting `generate_only=True` allows for checks to be disassembled when
+testing an estimator. This feature can be used to parameterize tests in pytest
+as follows::
+
+  from itertools import chain
+  import pytest
+  from sklearn.utils.estimator_checks import check_estimator
+  from sklearn.utils.estimator_checks import set_check_estimator_ids
+  from sklearn.svm import LinearSVC
+  from sklearn.linear_model import LogisticRegression
+
+  estimators = [LinearSVC, LogisticRegression]
+
+  @pytest.mark.parametrize(
+      'estimator, check',
+      chain.from_iterable(check_estimator(est, generate_only=True)
```

Comment:
Can `check_estimator` just return an iterator? It's a bit inconvenient that we have to chain manually

Reply:
This would work if `check_estimator` can be passed multiple estimators, and then generated everything on the spot.

Reply:
Well or we can make this example an example for just one estimator and it would be easier ;)
Or do both? First one estimator and then two, so we have the simple case first?